hooks/026-configure-plymouth.chroot

```diff
@@ -0,0 +1,141 @@
+#! /bin/sh -ex
+
+# Rewrite units for plymouth
+cat << 'EOF' > /lib/systemd/system/plymouth-start.service
```

Comment:
I feel like we should use drop-ins to tweak upstream units, that would be easier to maintain.

Reply:
Changed now to use overrides